### PR TITLE
Suggest GitHub tutorial edits

### PR DESCRIPTION
**Pull request by wslack**
Made some edits, and added two screenshots for non-tech people like moi who didn't understand that "folders" in GitHub are literally the same as "folders" in a computer. Thoughts?

### CODE REVIEW

**Comment by wslack** [comment]
Also removing the random cobras image.

File: _posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md [hunk 1]
```diff
@@ -93,7 +95,13 @@ I like to put all my GitHub projects in the same directory. So **the first thing
...
+Now we can start working with the website. GitHub is a system that stores files and records changes made to them using a piece of software called [Git](http://git-scm.com/), which allows multiple people to work on a program at the same time without getting in each others' way. If our program was a five paragraph essay, Git allows Corey to edit the introduction on one computer while Jesse edits the conclusion somewhere else. When both are done, they can move their edits into one copy while preserving the changes from each person.
```
Comment (mbland) [inline _posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md]: I'd replace "work on a program" with "make separate changes to a program".

File: _posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md [hunk 2]
```diff
@@ -93,7 +95,13 @@ I like to put all my GitHub projects in the same directory. So **the first thing
...
+
+If you've ever used "Track Changes" in word processing (where edits are highlighted to be approved or rejected), Git is like that, but with many more features. In software development, Git and other **version control** tools are helpful when looking for the origin of a bug. Let's say that some code isn't working, but 15 people made edits to it in the last week and you aren't sure where the bug is. With Git, you can load an old version of a program, test for the issue, and then "jump forward" through people's edits until you find the first one that also has the problem. That tells you the specific edit to search for the mistake.
+
+GitHub is a website that shows information from Git. For example, [this link](https://github.com/18F/18f.gsa.gov/commit/2df683fe1ef04f89ee77672d85ec35a11fec96d3) shows an spelling edit to this post by  Melody. It also allows easy discussion about proposed changes, or for people who don't have edit access to write-up problems they see as "issues." [Here](https://github.com/18F/18f.gsa.gov/issues/542) is an issue thread that we made specifically for comments on this post!
```
Comment (mbland) [inline _posts/2015-02-24-how-to-use-github-and-the-terminal-a-guide.md]: Nits: "a spelling edit", and an extra space before "Melody".

**Comment by mbland** [comment]
LGTM
cc: @melodykramer @gboone

**Pull request merged**